README.md

```diff
@@ -26,59 +26,87 @@ Unsure what to contribute? Check out the `good first issue` [tagged Github issue
 - [How to Mimic Boyfriend's Voice](https://youtu.be/YOrC9uQiK00)
-- [Writing Vocal Duets - LongestSoloEver](https://www.youtube.com/watch?v=nDPpO4fLiAM) - A beginner-friendly guide to writing duets in a Friday Night Funkin' song
+- [Writing Vocal Duets - LongestSoloEver](https://www.youtube.com/watch?v=nDPpO4fLiAM) - A beginner-friendly guide to writing duets in a Friday Night Funkin' song.
 - [6 Reasons your FNF Music Sucks - LongestSoloEver](https://www.youtube.com/watch?v=kela6mWtIlU) - Common mistakes when writing songs for FNF. Also a bit of hyperbole along the way.
-- [FNF Music Tutorial Playlist - LongestSoloEver](https://www.youtube.com/playlist?list=PLfb6KneL63QsQ58tj-RkDCHRmheAofPJj) - A playlist with all of LongestSoloEver's guides to writing FNF music (except the 2 listed above for some reason.) 
-
-- Chromatic Scale Guides
+- [FNF Music Tutorial Playlist - LongestSoloEver](https://www.youtube.com/playlist?list=PLfb6KneL63QsQ58tj-RkDCHRmheAofPJj) - A playlist with all of LongestSoloEver's guides to writing FNF music (except the 2 listed above for some reason).
+### Chromatic Scale Guides
   - [How to MAKE CHROMATICS (Friday Night Funkin) - bbpanzu](https://www.youtube.com/watch?v=a7SGu1fNthc)
   - [EASY FNF chromatic guide! (Friday Night Funkin') - StickyBM](https://youtu.be/PlSh_LJwQD0)
   - [How to make FNF Chromatics in Ableton - LongestSoloEver](https://www.youtube.com/watch?v=QCA6-N-pW_0)
 
 ## Programming
 ### Engines
-- [Base Friday Night Funkin' aka. Vanilla](https://github.com/FunkinCrew/funkin) - The original open source game by The Funkin' Crew.
+- [Friday Night Funkin'](https://github.com/FunkinCrew/funkin) - The original open source game by The Funkin' Crew.
 - [Psych Engine](https://github.com/ShadowMario/FNF-PsychEngine) - A fork of base game which includes new quality-of-life changes, performance improvements, and Lua-based mod tools. Popular and well-documented. Famous mods built with it include Hazy River and Whitty: Definitive Edition.
-- [Funkin' Forever Engine](https://github.com/BeastlyGabi/Funkin-Forever-Engine) - An archive of a rewrite of the game, originally written by Yoshubs. Includes performance reworks, HScript support, and more. Famous mods built with it include Hypno's Lullaby v2 and Mario Bros Funk Mix.
+- [Forever Engine](https://github.com/BeastlyGabi/Funkin-Forever-Engine) - An archive of a rewrite of the game, originally written by Yoshubs. Includes performance reworks, HScript support, and more. Famous mods built with it include Hypno's Lullaby v2 and Mario Bros Funk Mix.
+  - Has three available versions: [Legacy](https://github.com/BeastlyGabi/Funkin-Forever-Engine/tree/legacy), [1.0 (not finished)](https://github.com/BeastlyGabi/Funkin-Forever-Engine/tree/rewrite) and [Hybrid (mix of Legacy and 1.0)](https://github.com/BeastlyGabi/Funkin-Forever-Engine/tree/hybrid).
```

Comment:
Along with that I am unsure about adding information on all the different versions, I feel like people can look into that a bit themselves, or can just be featured part of the description text for the engine itself

also applies to Andromeda Engine below

Reply:
Okay, shall revert that as well.

Reply:
Done!